packages/cli/src/oclif/commands/env/unset.js

```diff
@@ -42,13 +42,27 @@ class UnsetEnvCommand extends BaseCommand {
     }
 
     const url = `/apps/${app.id}/versions/${version}/multi-environment`;
-
-    // currently, this returns nothing
-    // also, no need to cath errors here, since invalid keys don't get tripped over if the env var didn't exist in the first place
-    await callAPI(url, {
+    const requestOptions = {
       body: payload,
       method: 'POST',
-    });
+    };
+
+    if (this.flags.force) {
+      requestOptions.extraHeaders = {
+        'X-Force-Env-Var-Update': 'true',
+      };
+    }
+
+    try {
+      await callAPI(url, requestOptions);
+    } catch (e) {
+      if (e.statusCode === 409) {
+        this.error(
+          `App version ${version} is the production version. Are you sure you want to unset potentially live environment variables?` +
+            ` If so, run this command again with the --force flag.`,
+        );
+      }
```

Comment:
This will silently swallow any non-409 errors. Should we rethrow it?

```suggestion
      } else {
        throw e;
      }
```